src/Operations.ts

```diff
@@ -28,11 +28,12 @@ export class CreatePointer implements DSOperation {
 export class AssignPointer implements DSOperation {
     public type = "AssignPointer";
     public lhs: string;
-    public rhs: string;
-    public oldDestination: Node;
+    public nodeId: number;
```

Comment:
I think if we renamed `lhs` to `pointerName` and `nodeId` to `newNodeId`, the code in other files would be easier to understand.

Reply:
good idea